### PR TITLE
Dark Mode: Permissions container

### PR DESCRIPTION
**Pull request by darkwing**
Pretty straightforward

Connect

### CODE REVIEW

File: ui/components/app/permission-page-container/index.scss [hunk 1]
```diff
@@ -23,14 +23,6 @@
     }
   }
 
-  &__header {
```
Comment (mcmire) [inline ui/components/app/permission-page-container/index.scss:26]: This was unused, I gather?

**Comment by mcmire** [comment]
@darkwing Mind posting a video of switching between light/dark modes?

**Review by georgewrmarshall — APPROVED** [review]

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]
You're missing some colors

image
image

**Review by georgewrmarshall — DISMISSED** [review]

File: ui/components/app/permission-page-container/index.scss (hunk 1 above)
Comment (darkwing) [reply ui/components/app/permission-page-container/index.scss:26]: Yep!  Couldn't find it.

**Review by georgewrmarshall — DISMISSED** [review]

**Comment by darkwing** [comment]
@GuillaumeRx Are you sure?  I'm seeing good stuff:
perms

**Review by GuillaumeRx — DISMISSED** [review]
There is something weird with the storybook build, the colors are updated.

**Review by GuillaumeRx — APPROVED** [review]

**Review by georgewrmarshall — APPROVED** [review]

**Pull request merged**